parsl/executors/high_throughput/mpi_prefix_composer.py

```diff
@@ -0,0 +1,102 @@
+import logging
```

Comment:
architecturally, this feels to me like a weird reimplementation of the parsl.launchers code. I'm not sure what the best approach though, but two enemy implementations of a launcher API, at least on first glance, feels bad to me.

Reply:
I do not like the duplication either, but it made some sense to separate it given that launchers are designed to launch one manager per node and no more, while mpi_launchers have to be flexible to support a wider range.

Reply:
Let's plan on reworking this MPI launcher to integrate into the existing launchers. There are some troublesome bits from how the launchers are limited to just launching managers one per node.